lesson21/index.js

```diff
@@ -0,0 +1,239 @@
+    const tableHead = createTableHead(titleNames);
+    const tableBody = createTableBody(data, titleKeys);
+
+    parent.appendChild(table).appendChild(tableHead).after(tableBody);
+};
+
+
+const createTableHead = titlesData => {
+    const tableHead = document.createElement("thead");
+    const tr = createElementWithClassName("tr", "table__row");
+    const fragment = document.createDocumentFragment();
+    
+    for(let i = 0; i < titlesData.length; i++){
+        const th = createElementWithClassName("th", "table__title js-table-title");
+        th.textContent = titlesData[i];
+        fragment.appendChild(th);
+    }
+    tableHead.appendChild(tr).appendChild(fragment);
+
+    return tableHead;
+};
+
+const createTableBody = (data, keys) => {
+    const tableBody = document.createElement("tbody");
+    const fragment = document.createDocumentFragment();
+
+    for(let i = 0; i < data.length; i++){
+        const tr = createElementWithClassName("tr", "table__row js-table-row");
+        const td = createTableContents(data[i], keys);
+        fragment.appendChild(tr).appendChild(td);
+    }
+    tableBody.appendChild(fragment);
+
+    return tableBody;
+};
+
+const createTableContents = (data, keys) => {
+    const fragment = document.createDocumentFragment();
+
+    for(let i = 0; i < keys.length; i++){
+        const td = createElementWithClassName("td", "table__contents js-table-contents");
+        td.textContent = data[keys[i]];
+        fragment.appendChild(td);
+    }
+
+    return fragment;
+};
+
+const createSortButtons = () => {
+    const ul = createElementWithClassName("ul", "sort-buttons");
+    ul.id = "js-sort-buttons";
+
+    const buttonConfig = [
+        { src: "../img/icon-both.svg", alt: "idを昇順に並び替える", dataSet: "default"},
+        { src: "../img/icon-asc.svg", alt: "idを降順に並び替える", dataSet: "asc"},
+        { src: "../img/icon-desc.svg", alt: "idを順不同に並び替える", dataSet: "desc"}
+    ]
+
+    const fragment = document.createDocumentFragment();
+    buttonConfig.forEach((item, index) => {
+        const li = createElementWithClassName("li", "sort-buttons__item js-sort-button-item");
+        const button = createElementWithClassName("button", "sort-buttons__btn");
+        const img = createElementWithClassName("img", "sort-buttons__img");
+
+        li.dataset.index = index;
+        li.dataset.buttonStatus = item.dataSet;
+        img.src = item.src;
+        img.alt = item.alt;
+        fragment.appendChild(li).appendChild(button).appendChild(img);
+    })
+    ul.appendChild(fragment);
+    return ul;
+};
+
+const renderSortButton = () => {
+    const thArray = [...document.querySelectorAll(".js-table-title")];
+    thArray[0].id = "js-set-button";
+    document.getElementById("js-set-button").insertAdjacentElement('beforeend', createSortButtons());
+};
+
+const setButtonForInitDisplay = () => {
+    document.querySelector('[data-button-status="default"]').classList.add("is-active");
+};
```

Comment:
IMO
If the body of the function is one sentence, {} can be omitted as shown below.
```suggestion
const setButtonForInitDisplay = () => document.querySelector('[data-button-status="default"]').classList.add("is-active");
```

Reply:
Thank you for your review!
That's right! I hadn't noticed.
I had omitted the {} in the function in other parts of the code, so I adjusted it.
Thank you!
[fe9a5c5](https://github.com/haru-programming/JavaScript-lessons/pull/26/commits/fe9a5c5b36f9302ef280e5b5ffe616315102cacb)